### PR TITLE
Allow filtering participations and investments based on time elapsed

### PR DESCRIPTION
**Pull request by triceo**
This is the final change to the strategy for the foreseeable future. It allows to filter secondary marketplace (both purchase and sale) based on how many payments have already been made on the participation/investment. 

Rationale behind this change: data from some P2P platforms show that loans are most vulnerable within their first 24 instalments. Delinquence goes down throughout the entire period and is nearly negligible after that. Users should be allowed to reflect that in their strategies.

Won't merge before there's support in the web tool. No rush.

### CODE REVIEW

**Comment by jhrcek** [comment]
Ok, will take a look sometimes this week..

File: robozonky-strategy-natural/src/main/antlr4/imports/CommonFilters.g4 [hunk 1]
```diff
@@ -99,6 +99,52 @@ relativeTermConditionRangeClosedRight returns [RelativeLoanTermCondition result]
...
+    ) ' splátek'
+;
+
+elapsedTermConditionRangeOpen returns [ElapsedLoanTermCondition result]:
+    IS min=intExpr UP_TO max=intExpr
+    { $result = new ElapsedLoanTermCondition($min.result, $max.result); }
+;
+
+elapsedTermConditionRangeClosedLeft returns [ElapsedLoanTermCondition result]:
+    'více než' min=intExpr
```
Comment (jhrcek) [inline robozonky-strategy-natural/src/main/antlr4/imports/CommonFilters.g4]: Any reason not to use MORE_THAN (as opposed to literal string)?

Comment (triceo) [reply robozonky-strategy-natural/src/main/antlr4/imports/CommonFilters.g4]: Exactly. There was no way to make a decent Czech sentence using the standard MORE_THAN.

**Comment by jhrcek** [comment]
I implemented support for these new conditions as you requested.

Few more notable changes:
1. removed tooltip about captcha
2. Since the filter creation modal starts to be a bit crowded, I tried to make it a bit more readable by drawing border around each filter condition that user enables - I find the form a bit more readable with that. WDYT?
screenshot from 2017-11-10 13-54-16

Check the updated version [on my gh pages](http://janhrcek.cz/natural-strategy-setup/) and let me know if you find any issues / have suggestions for improvement.

**Comment by jhrcek** [comment]
One more thing: once you give me some feedback I'll close https://github.com/RoboZonky/robozonky/pull/162 and reopen new PR with all changes included.

**Comment by triceo** [comment]
Very nice! For the new border, I'd perhaps just increase the bottom margin so that there is some more white space between the bottom border and the checkbox directly below. (So that it looks like the white space above the top border.

**Pull request merged**